demos/calendar/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>jQuery UI Calendar Demos</title>
+</head>
+<body>
+
+<ul>
+	<li><a href="default.html">Default functionality</a></li>
+	<li><a href="buttonbar.html">Display button bar</a></li>
+	<li><a href="dropdown-month-year.html">Display month &amp; year menus</a></li>
+	<li><a href="localization.html">Localize calendar</a></li>
+	<li><a href="min-max.html">Restrict date range</a></li>
+	<li><a href="multiple-months.html">Display multiple months</a></li>
+	<li><a href="other-months.html">Dates in other months</a></li>
+	<li><a href="show-week.html">Show week of the year</a></li>
+</ul>
```

Comment:
`multiple-calendars.html` isn't listed, but it looks like it's just a duplicate of `multiple-months.html`, so we should delete that file.